### PR TITLE
feat: Mureva EVlink Smart socket outlet

### PR DESCRIPTION
**Pull request by acesyde**
Image : https://github.com/Koenkk/zigbee2mqtt.io/pull/2865

### CODE REVIEW

File: src/devices/schneider_electric.ts [hunk 1]
```diff
@@ -1546,6 +1546,31 @@ const definitions: Definition[] = [
             await reporting.currentSummDelivered(endpoint, {min: 60, change: 1});
         },
     },
+    {
+        zigbeeModel: ['EVSCKT/OUTLET/1'],
+        model: 'MUR36014',
+        vendor: 'Schneider Electric',
+        description: 'Mureva EVlink Smart socket outlet',
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.power_on_behavior],
```
Comment (Koenkk) [inline src/devices/schneider_electric.ts:1554]: Can you try using modern extend here? [example](https://github.com/Koenkk/zigbee-herdsman-converters/blob/d39fb62c7af49084f23c106691eeb683320736b2/src/devices/gs.ts#L49)

Comment (btpemercier) [reply src/devices/schneider_electric.ts]: I think it's okay now

Comment (acesyde) [reply src/devices/schneider_electric.ts]: I think it's okay now

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**